src/blant.c

```diff
@@ -917,37 +918,40 @@ int main(int argc, char *argv[])
         if (strncmp(optarg, "n", 1) == 0 || strncmp(optarg, "N", 1) == 0) {
             _windowRep_limit_neglect_trivial = true; optarg += 1;
         }
-		if (strncmp(optarg, "DEG", 3) == 0) { 
-			_windowRep_limit_method = WINDOW_LIMIT_DEGREE; optarg += 3; 
+		if (strncmp(optarg, "DEG", 3) == 0) {
+			_windowRep_limit_method = WINDOW_LIMIT_DEGREE; optarg += 3;
 		}
-		else if (strncmp(optarg, "EDGE", 4) == 0) { 
-			_windowRep_limit_method = WINDOW_LIMIT_EDGES; optarg += 4; 
+		else if (strncmp(optarg, "EDGE", 4) == 0) {
+			_windowRep_limit_method = WINDOW_LIMIT_EDGES; optarg += 4;
 		}
-		else 
+		else
 			Fatal("Unrecognized window limiting method specified. Options are: -l{DEG}{EDGE}{limit_num}\n");
 		_numWindowRepLimit = atoi(optarg);
         if (!_numWindowRepLimit) {_numWindowRepLimit = 10; _numWindowRepArrSize = _numWindowRepLimit;}
 		_windowRep_limit_heap = HeapAlloc(_numWindowRepLimit, asccompFunc, NULL);
 		break;
-	case 'n': numSamples = atoi(optarg);
+    case 'n': numSamples = atoi(optarg);
 	    if(numSamples < 0) Fatal("numSamples must be non-negative\n%s", USAGE);
 	    break;
-    case 'K': _KS_NUMSAMPLES = atoi(optarg); 
+    case 'K': _KS_NUMSAMPLES = atoi(optarg);
     	break;
-    case 'e': 
-        _GRAPH_GEN_EDGES = atoi(optarg); 
+    case 'e':
+        _GRAPH_GEN_EDGES = atoi(optarg);
         windowRep_edge_density = atof(optarg);
         break;
-    case 'g': 
+    case 'g':
         if (!GEN_SYN_GRAPH) Fatal("Turn on Global Variable GEN_SYN_GRAPH");
-        _GRAPH_GEN = true; 
+        _GRAPH_GEN = true;
         if (_genGraphMethod != -1) Fatal("Tried to define synthetic graph generating method twice");
         else if (strncmp(optarg, "NBE", 3) == 0) _genGraphMethod = GEN_NODE_EXPANSION;
         else if (strncmp(optarg, "MCMC", 4) == 0) Apology("MCMC for Graph Syn is not ready");  // _genGraphMethod = GEN_MCMC;
         else Fatal("Unrecognized synthetic graph generating method specified. Options are: -g{NBE|MCMC}\n");
         break;
 	case 'u': UNIQ_GRAPHLETS = false;
 	    break;
+    case 'M': multiplicity = atoi(optarg);
+        if(multiplicity < 0) Fatal("multiplicity must be non-negative\n%s", USAGE);
```

Comment:
It's best to put the USAGE message *first*, and the error message at the bottom, otherwise the user may not see the error as it'll scroll off the screen.